### PR TITLE
Add Subscribe page

### LINKED ISSUE #244: Add direct to route to website to allow us to subscribe
## Before you `start work`

Please read our contribution [guidelines](https://docs.marketprotocol.io/#contributing) and if there is a bounty involved  please also see [here](https://docs.marketprotocol.io/#gitcoin-and-bounties)

If you have ongoing work from other bounties with us where funding has not been released, please do not pick up a new issue.  We would like to involve as many contributors as possible and parallelize the work flow as much as possible.

Please make sure to comment in the issue here immediately after starting work so we know your plans for implementation and a timeline.

Please also note that in order for work to be accepted, all code must be accompanied by test cases as well.

### User Story
As a user I want to be able to have a direct link to the subscribe modal for the newsletter.

### Why Is this Needed?
*Summary*: More subscribers

### Description
*Type*: Feature

### Current Behavior
To access the modal you have to click on `Join our Newsletter`

image

### Expected Behavior
marketprotocol.io/subscribe would direct to main site with modal launched

### Reproduction
see https://marketprotocol.io/

### Solution

*Summary*: add direct route

### Definition of Done
- [ ] add new route 
- [ ] add needed tests

### PR DESCRIPTION
**Pull request by arun1595**
##### Description

Adds a new Subscribe page

##### Checklist

- [x] Linter status: 100% pass
- [x] Changes don't break existing behavior
- [x] Tests coverage hasn't decreased

##### Testing

I've added a basic test to make sure my code works and all other tests pass.

##### Refers/Fixes

Fixes #244

### CODE REVIEW

**Comment by eswarasai** [comment]
@arun1595 -- Any update on this one? Let me know if you need any help here. Thanks!

**Comment by arun1595** [comment]
@eswarasai Done. PTAL and let me know if I missed something.

Thanks.

File: src/containers/Subscribe.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import { withSiteData } from 'react-static';
+import { HomeComponent } from '@containers/Home';
+
+export default withSiteData(() => <HomeComponent subscriptionPopUpVisible />);
```
[the remark this answers is not in the document]
Comment (eswarasai) [reply src/containers/Subscribe.tsx]: @arun1595 -- There's a TSLint check [`jsx-boolean-value`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md). Not sure why `website` is not showing up lint errors. Probably the recommended `tslint.json` is updated or something. Need to check it though.

File: src/containers/Home.tsx [hunk 2]
```diff
@@ -10,7 +10,13 @@ import BuildDapp from '@components/BuildDapp';
 import Cta from '@components/Cta';
 import ExplainerCta from '@components/ExplainerCta';
 
-export const HomeComponent = ({ enableVideo }) => (
+export const HomeComponent = ({
+  subscriptionPopUpVisible,
```
[the remark this answers is not in the document]
Comment (arun1595) [reply src/containers/Home.tsx]: `HomeComponent` is a stateless component, what do you refer by an interface for `HomeComponentState`?

**Comment by arun1595** [comment]
@eswarasai I've added a test for `Subscribe` container and an interface for `HomeComponentProps`. PTAL :slightly_smiling_face:

File: src/containers/Home.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (eswarasai) [reply src/containers/Home.tsx]: @arun1595 -- You're right. My bad. Confused it with `props` and `state` being passed. Thanks for pointing it out 👍

**Comment by eswarasai** [comment]
@arun1595 -- Thanks for the quick turn around on the requested changes. Weird that the coverage percentage is still decreased by 0.5%. Anyway I'm going ahead and merging this PR.

**Pull request merged**